src/domain/payments/ln-fees.ts

```diff
@@ -45,8 +45,14 @@ export const LnFees = () => {
     priceRatio: WalletPriceRatio
     senderWalletCurrency: WalletCurrency
   }) => {
-    const calculatedMaxFeeAmount = maxProtocolAndBankFee(btcPaymentAmount)
+    const calculatedMaxFeeAmount =
+      senderWalletCurrency === WalletCurrency.Btc
+        ? maxProtocolAndBankFee(btcPaymentAmount)
+        : priceRatio.convertFromUsd(
+            maxProtocolAndBankFee(priceRatio.convertFromBtc(btcPaymentAmount)),
+          )
```

Comment:
not sure if I am not undestanding correctly but... why dont use convertFromBtc? 
```suggestion
        : priceRatio.convertFromBtc(
            maxProtocolAndBankFee(btcPaymentAmount),
          )
```